DTML.EduBot/LessonPlan/lesson_plan.json

```diff
@@ -2,24 +2,58 @@
     "lessons":[  
         {  
           "lesson_title":"A few words every day",
-          "topics":[  
-              {  
-                "question":"Lets start off with 10 questions. Please identify the image below:",
-                "image_url":"https://images.contentful.com/6m9bd13t776q/6KmARX9mUwsuG4O8YsI8KY/4f68bd0d9d9d4d8a1ab5307f0fa59679/which-baby-names-stand-time-test-2160x1200.jpg",
+          "topics":[
+            {
+              "question": "Lets start off with 10 questions. Please identify the image below:",
+              "image_url": "https://images.contentful.com/6m9bd13t776q/6KmARX9mUwsuG4O8YsI8KY/4f68bd0d9d9d4d8a1ab5307f0fa59679/which-baby-names-stand-time-test-2160x1200.jpg",
+              "answer_options": [
+                "A baby",
+                "A robot",
+                "A computer"
+              ],
+              "correct_answer": "A baby",
+              "correct_answer_bot_response": "Correct! Now, can you type the word?",
```

Comment:
Maybe this is way too many options. Let's have a default value for this in code. people can override if they want.